src/external_cli/cargo/metadata.rs

```diff
@@ -14,12 +14,12 @@ pub(crate) fn command() -> Command {
     command
 }
 
-/// Try to obtain the Cargo metadata of this pacakge.
+/// Try to obtain the Cargo metadata of this package.
 pub(crate) fn metadata() -> anyhow::Result<Metadata> {
     metadata_with_args::<[&str; 0], &str>([])
 }
```

Comment:
This should probably specify the `--format-version 1` flag. Calling `cargo metadata` without it causes a warning, and pinning the format version prevents breaking changes in the future.

Reply:
`--format-version 1` is already passed when creating the command :)